.github/workflows/sync-python-releases.yml

```diff
@@ -20,8 +20,8 @@ jobs:
       - uses: hynek/setup-cached-uv@v1
       - name: Sync Python Releases
         run: |
-          uv run --isolated -- fetch-download-metadata.py
-          uv run --isolated -- template-download-metadata.py
+          uv run --no-workspace -- fetch-download-metadata.py
+          uv run --no-workspace -- template-download-metadata.py
```

Comment:
A little tedious...

Reply:
Doesn't inline metadata imply isolation? `uv run fetch-download-metadata.py` works for me